graphql/auth.graphql

```diff
@@ -52,9 +52,11 @@ input RegisterInput {
     first_name: String! @rules(apply: ["required", "string"])
     last_name: String! @rules(apply: ["required", "string"])
     phone_number: String! @rules(apply: ["required", "string"])
+    password:String
```

Comment:
Also add a confirm password field, and add validation to ensure the following:

1. The password and confirm_password fields are the same
2. The password has a minimum of 6 characters.

Read through the lighthouse-php validationn docs [here](https://lighthouse-php.com/4.10/security/validation.html#validating-arguments)